apps/dapp-console-api/src/constants/envVars.ts

```diff
@@ -72,6 +72,16 @@ const envVarSchema = z.object({
   CB_VERIFICATION_ATTESTER: z
     .string()
     .default('0x357458739F90461b99789350868CD7CF330Dd7EE'),
+  FAUCET_V1_CONTRACT_ADDRESS: z.string(),
+  FAUCET_V1_AUTH_ADMIN_WALLET_PRIVATE_KEY: z
```

Comment:
nit: can leave the `V1` out since there is no other version of the faucet in this repo: (e.g. `FAUCET_CONTRACT_ADDRESS` and `FAUCET_AUTH_ADMIN_WALLET_PRIVATE_KEY`)